sdk/helper/keysutil/policy.go

```diff
@@ -220,6 +229,10 @@ type KeyEntry struct {
 	// This is deprecated (but still filled) in favor of the value above which
 	// is more precise
 	DeprecatedCreationTime int64 `json:"creation_time"`
+
+	// NOTE: Do we really need this field? If so, having it here or in Policy?
+	// Useful for validation in operations in case we only have public key
+	PublicKeyImported bool `json:"public_key_imported"`
```

Comment:
Like this comment :-) 

Perhaps this could be a helper method off of this `KeyEntry` struct that returns the value appropriately (`func ... { return RSAKey == nil && EC_... == nil`?)